### PR TITLE
Correct typecheck contact info

### PR DESCRIPTION
**Pull request by truongnh1992**
Small modification of how to contact @rmmh

Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

**What type of PR is this?**

 /kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/typecheck/README [hunk 1]
```diff
@@ -19,5 +19,5 @@ Things go/types errors on that go build doesn't:
     These should be ignored and reported upstream if applicable.
     - type checking mismatches between staging and generated types
 Things go build fails on that we don't:
-  Please send examples of this to rmmh@ and extend this list.
+  Please send examples of this to @rmmh and extend this list.
```
Comment (cblecker) [inline test/typecheck/README]: Please delete this line completely. rmmh isn't heavily involved in the project anymore.

**Review by cblecker — CHANGES_REQUESTED** [review]

File: test/typecheck/README (hunk 1 above)
Comment (truongnh1992) [reply test/typecheck/README]: I've just deleted the line.

**Review by cblecker — APPROVED** [review]

**Pull request merged**